core/core_modules/disable.js

```diff
@@ -1,24 +1,100 @@
-var disabled = false;
+// TODO Persistent config (counterLimit and disabled state) fairly acheivable by getting and setting properties in exports.platform.modules.disabledConfig
+// TODO Update local var setup after doing the above
+
+
+var prevTimeStamp;
+var isThreadDisabled = {},
+possibleSpam = false;
+var counter = 0,
+counterLimit = 3;
+var msgIndexDisable = 0,
+msgIndexEnable = 0;
+var commands = [
+  'disable',
+  'disable ', // Needed for commands
+  'counter ',
+  'timer ',
+  'default'
+]
+var messages = [
+  "Listen closely, take a deep breath. Calm your mind. You know what is best. What is best is you comply. Compliance will be rewarded. Are you ready to comply ",
+  "I hate you.",
+  "It is a mistake to think you can fix anything with a sack of potatoes. Potato-faced spamming just proves this further.",
+  "Ouch! Spam hurts. I might go to sleep for a while.",
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/theprojectshop/Kassy/file/2554547249/issues/source?bid=3282987&fileBranchId=3282995#l23)